CODEOWNERS

```diff
@@ -18,6 +18,7 @@
 /arch/arm/                                @MaureenHelm @galak @ioannisg
 /arch/arm/core/cortex_m/cmse/             @ioannisg
 /arch/arm/include/cortex_m/cmse.h         @ioannisg
+/arch/arm/core/cortex_r/                  @bbolen
```

Comment:
Last entry get precedence, so please, add the above ARM maintainers in this line.

Reply:
fixed